File: apps/mobile/lib/socket/events.dart
// Mirror of apps/server/src/sockets/events.js — keep these strings in sync
// with the server so the contract never drifts.
class Events {
  static const joinSession = 'session:join';
  static const leaveSession = 'session:leave';
  static const sessionMembers = 'session:members';
  static const closeSession = 'session:close';

  static const startRecording = 'recording:start';
  static const stopRecording = 'recording:stop';
  static const recordingStarted = 'recording:started';
  static const recordingStopped = 'recording:stopped';
}

File: apps/server/src/sockets/events.js
// Socket.IO event names shared by the web and mobile clients.
export const EVENTS = {
  JOIN_SESSION: "session:join",
  LEAVE_SESSION: "session:leave",
  SESSION_MEMBERS: "session:members",
  CLOSE_SESSION: "session:close",

  START_RECORDING: "recording:start",
  STOP_RECORDING: "recording:stop",
  RECORDING_STARTED: "recording:started",
  RECORDING_STOPPED: "recording:stopped",
};

export const DEFAULT_RECORDING_BUFFER_MS = 50;
